quickstarts/oma-bofu/config.yml

```
@@ -0,0 +1,50 @@
+# Name of the quickstart (required)
+# sets the URL name of the quickstart on public I/O
+name: customer-experience-bottom-funnel-analysis
+
+# Displayed in the UI (required)
+title: Customer Experience Bottom of the funnel analysis 
+
+# Long-form description of the quickstart (required)
+description: |
+  Bottom of the funnel analysis uses cart abandonment reduction techniques and applies them to any user journey where conversion (completing the user journey) matters.  
+ 
+  For more information view the [implementation guide](https://docs.newrelic.com/docs/new-relic-solutions/observability-maturity/customer-experience/bottom-funnel-analysis-customer-journey-guide)
+
+# Displayed in search results and recommendations. Summarizes a quickstarts functionality.
+summary: |
+  Use cart abandonment reduction techniques to improve completion rates of any user journey 
+
+# Support level: New Relic | Verified | Community (required)
+level: New Relic 
+
+# Authors of the quickstart (required)
+authors:
+  - Kim Hickey 
+
+# Keywords for filtering / searching criteria in the UI
+keywords:
+  - observability maturity
+  - proactive observability
+  - BOFU
+  - customer experience
+  - browser
+  - core web vitals
+  - ajax
+  - conversion rate
+  - conversions
+  - synthetics
+
+# Documentation references
+  - name: Install README
+    description: |
+      Guides you on how to customize the dashboard once you've installed it
+    url: https://github.com/newrelic/newrelic-quickstarts/tree/main/quickstarts/oma-bofu#readme
+  - name: Implementation guide
+    description: |
+      Explains what bottom of the funnel analysis is and how to apply it
+    url: https://docs.newrelic.com/docs/new-relic-solutions/observability-maturity/customer-experience/bottom-funnel-analysis-customer-journey-guide
```

Looks like these are just missing their heading
```suggestion
documentation:
  - name: Install README
    description: |
      Guides you on how to customize the dashboard once you've installed it
    url: https://github.com/newrelic/newrelic-quickstarts/tree/main/quickstarts/oma-bofu#readme
  - name: Implementation guide
    description: |
      Explains what bottom of the funnel analysis is and how to apply it
    url: https://docs.newrelic.com/docs/new-relic-solutions/observability-maturity/customer-experience/bottom-funnel-analysis-customer-journey-guide
```

---

Whoops ..